Riot/Modules/MatrixKit/Models/Room/MXKRoomBubbleCellData.m

```diff
@@ -103,6 +107,18 @@ - (void)dealloc
     bubbleComponents = nil;
 }
 
+- (void)refreshProfileWithRoomDataSource:(MXKRoomDataSource *)roomDataSource2
+{
+    if (self.events.firstObject)
+    {
+        MXEvent* firstEvent = self.events.firstObject;
+        senderDisplayName = [roomDataSource.eventFormatter senderDisplayNameForEvent:firstEvent
+                                                                       withRoomState:roomDataSource2.roomState];
```

Comment:
Shouldn this maybe use its internal roomDataSource and only receive the `roomState` from the outside? I find it a bit weird for it to rely on 2 different roomDataSource instances. Perhaps we can make it a bit more generic with a public `setRoomState`? 🤔